.github/workflows/ci.yml

```diff
@@ -377,6 +377,6 @@ jobs:
       - name: Maven Install
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl presto-kudu
+          ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl presto-spark-launcher,presto-spark-package,presto-spark-testing
```

Comment:
Ooops. Sorry hope that wasn't me.

Reply:
It doesn't really matter who and when, docker based tests have been broken anyway up until recently =D